### DISCUSSION TITLE
I keep getting "no_stream_exist" error when creating a conference room

### DISCUSSION THREAD

**Discussion by itslenny** [ant-media/Ant-Media-Server#6533] [Q&A]
I've created a very simple conference call app following the examples found in `conference.html` and `conference-room.html` and every time I call `play()` I get the `no_stream_exist` error. Everything starts correctly and works, but this error keeps persisting and I can't figure out how to get it to not happen. 

I'm using the JavaScript SDK and using the following code

```JavaScript
// in 'initialized' callback
rtcAdapter.joinRoom(roomId, streamId);

// in 'joinedTheRoom' callback
rtcAdapter.publish(streamId);
rtcAdapter.play(roomId, '', roomId)
```

I also tried adding an interval that calls `rtcAdapter.getRoomInfo(roomId, streamId)` every 5 seconds, and calling `rtcAdapter.play(roomId, '', roomId)` the first time the `roomInformation` callback fired, but it didn't make any difference (this idea was taken from the code in the `conference-room.html` example, but it doesn't seem to change anything.

The `streamId` that is in the `no_stream_exist` error is the roomId provided when joining the room / calling play.

The error does not happen if I don't call `play()`, but I also don't get any remote tracks/streams if I don't call play.

I've also tried swapping `roomId` for `streamId` in both places in the play call. Changing the first parameter results in an RTC error and connection failure. Changing the second one doesn't seem to change anything.

**Comment by rahul7827** [comment]
Hello @itslenny 

Can you please share the code snippet you are using, I am not active in the community so not aware about the latest update in the SDK.
I can help you with the fix by recreating the scenario at my end. Also share the media server version and JS SDK version that you are using.
